devops/aws/deploy-playground-playbook.yml

```diff
@@ -106,16 +106,28 @@
         caddy_systemd_capabilities_enabled: true
         caddy_update: false
 
+    - name: Set endpoints
+      set_fact:
+        all_services: |
+          The services should now be accesible at:
```

Comment:
Perhaps it would be best to output this in a format that is easier to parse programmatically. like an env file or even better json (why not both) or both.
Exposing this information over one more endpoint in caddy would also support idea here: https://github.com/Joystream/pioneer/issues/2167#issuecomment-1029636565